src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,39 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
```

Comment:
- Avoid magic values such as 100, etc. Create constants
- Health is 100 by default, but I think it should be possible to path another health value if needed

Reply:
Somewhy in vs code it's said that the commit was sent but here I can't see it. Odd